src/services/asset-inventory/cloud-service/cloud-service-detail/modules/cloud-service-history-detail/CloudServiceHistoryChangesTab.vue

```diff
@@ -77,11 +79,15 @@ export default defineComponent({
             type: Object as PropType<CloudServiceHistoryItem>,
             default: () => ({}),
         },
+        selectedKeyName: {
+            type: String,
+            default: '',
+        },
     },
     setup(props) {
         const state = reactive({
-            selectedKeyMenu: '',
-            keyMenus: computed(() => props.selectedHistoryItem?.diffItems?.map((d, index) => ({ label: d.key, name: `${d.key}-${index}`, updateType: d.type })) ?? []),
+            keyMenus: computed(() => props.selectedHistoryItem?.diffItems?.map(d => ({ label: d.key, name: d.key, updateType: d.type })) ?? []),
+            selectedKeyMenu: props.selectedKeyName || props.selectedHistoryItem?.diffItems?.at(0)?.key || '',
```

Comment:
selectedKeyMenu는 computed 속성으로, keyMenus 안에서 찾는 게 좋을듯요!
이런 식으로..?

```suggestion
            selectedKeyMenu: computed(() => state.keyMenus.find(d => d.name === props.selectedKeyName) || ...state.keyMenus?.[0] || undefined),
```

Reply:
context menu의 selected에 어떤 타입이 들어가는지 가물해서...확인부탁드려요!

Reply:
selectedKeymenu 를 computed 로 감싸면 readonly 에러가 나요
selected의 타입은  ```Array as PropType<MenuItem[]>``` 인 것으로 보아 .. 코드 상 맞게 들어간 것 같습니다

Reply:
이 부분은 일단 머지하고 세미 QA 진행 후 고치겠습니다

Reply:
아!!!!! 맞다;; 저기 selected라...ㅋㅋㅋㅋㅋㅋㅋ죄송ㅋㅋㅋㅋㅋㅋㅋ넵넵